libs/ui/elements/src/lib/metadata-info/linkify.directive.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @angular-eslint/directive-selector */
+import { Directive, ElementRef, Renderer2, OnInit } from '@angular/core'
+
+@Directive({
+  selector: '[gnUiLinkify]',
+})
+export class GnUiLinkifyDirective implements OnInit {
+  constructor(private el: ElementRef, private renderer: Renderer2) {}
+
+  ngOnInit() {
+    setTimeout(() => {
+      this.processLinks()
+    }, 0)
+  }
+
+  private processLinks() {
+    const container = this.el.nativeElement
+    const linkRegex = /(\bhttps?:\/\/\S+\b)/g
+
+    const nodes = Array.from(container.childNodes)
+    nodes.forEach((node) => {
+      if (node instanceof Text) {
+        const textNode = node as Text
+        const linkified = this.linkifyText(textNode.nodeValue)
+        const span = this.renderer.createElement('span')
+        span.innerHTML = linkified
+        container.insertBefore(span, textNode)
+        container.removeChild(textNode)
+      }
+    })
+  }
+
+  private linkifyText(text: string): string {
+    return text.replace(/(\bhttps?:\/\/\S+\b)/g, (match) => {
+      return `<a href="${match}" target="_blank"
+                  class="contact-website text-primary cursor-pointer hover:underline transition-all">
```

Comment:
For point 1, well noted, I'll fix that.
For points 2-4, i will remove those classes.
For point 3, I actually had some trouble rendering the icon (used to render the text "open_in_new" instead of the icon) and these classes probably come from the solution I found online. I will try to remove them as well if they are indeed useless.

Reply:
I mean, maybe I'm also missing something here... the `mat-icon` component is a bit weird sometimes. Let me know if that works out

Reply:
No, you were right. I removed them and it works :)